src/StreamService.Business/StreamService.Business.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>

```

Comment:
same comment as in StreamService.Broker.csproj

Reply:
it was generated by vs and later fixed